lib/screens/components/hero_card.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+
+import '../../domain/models/marvel_hero.dart';
+import '../../themes/app_colors.dart';
+import '../../utils/constants/routes.dart';
+
+class HeroCard extends StatelessWidget {
+  const HeroCard({required this.hero, super.key});
+  final MarvelHero hero;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(16, 8, 16, 0),
+      child: Card(
+        color: AppColors.redCard,
+        clipBehavior: Clip.antiAlias,
+        elevation: 10,
+        shadowColor: AppColors.dark,
+        child: Ink(
+          width: 100,
+          child: InkWell(
+            borderRadius: BorderRadius.circular(10),
+            onTap: () =>
+                Navigator.pushNamed(context, Routes.details, arguments: hero),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Expanded(
+                  child: Image.network(
+                    hero.image,
+                    fit: BoxFit.cover,
+                    height: 120,
+                  ),
+                ),
```

Comment:
O que você fez não está errado, mas como sugestão/aprendizado, segue uma alternativa útil para alguns casos em que só a proporção importa.
```suggestion
                  child: AspectRatio(
                    aspectRatio: 1,
                    child: Image.network(
                      hero.image,
                      fit: BoxFit.cover,
                      height: 120,
                    ),
                  ),
                ),
```